src/fcmcts/fcmcts.jl

```diff
@@ -212,7 +212,7 @@ function insert_node!(tree::FCMCTSTree, planner::FCMCTSPlanner, s)
     end
 
     # NOTE: Doing state-dep actions here the JointMDP way
-    state_dep_jtactions = vec(map(collect, Iterators.product((get_agent_actions(planner.mdp, i, si) for (i, si) in enumerate(s))...)))
+    state_dep_jtactions = vec(map(collect, Iterators.product((agent_actions(planner.mdp, i, si) for (i, si) in enumerate(s))...)))
```

Comment:
As far as I can tell, this is the only place we are deviating from vanilla MCTS on an unfactored MMDP. I suppose it is worth maintaining separately for this alone

Reply:
Right!

Reply:
Actually, yeah if we do this then we shouldn't need FCMCTS anymore at all!
What we can do for now is to remove `FCMCTS` from this repo so it does not hold up the rest of it. Later on, we can fix our own baseline scripts to use vanilla MCTS.